### ISSUE TITLE
unexpected_request_content_type

### ISSUE THREAD

**Issue by epcim** [zbentley/classeur-downloader#2]
Would you be able to fix the content type:

Not sure why it point to option arg parser, but it sucesfully goes to downloader to store files:

``` npm/lib/node_modules/classeur-downloader/bin/cldownload.js:11
        if (error) throw error;
                   ^

Error: Server error (400): unexpected_request_content_type
    at ServerError (/home/pmichalec/hg2g/apps/npm/lib/node_modules/classeur-downloader/node_modules/classeur-api-client/lib/errors.js:27:3)
    at Request.<anonymous> (/home/pmichalec/hg2g/apps/npm/lib/node_modules/classeur-downloader/node_modules/classeur-api-client/classeur-api-client.js:212:27)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request.mixin._fireSuccess (/home/pmichalec/hg2g/apps/npm/lib/node_modules/classeur-downloader/node_modules/restler/lib/restler.js:223:12)
    at /home/pmichalec/hg2g/apps/npm/lib/node_modules/classeur-downloader/node_modules/restler/lib/restler.js:161:20
    at IncomingMessage.parsers.json (/home/pmichalec/hg2g/apps/npm/lib/node_modules/classeur-downloader/node_modules/restler/lib/restler.js:415:9)
    at Request.mixin._encode (/home/pmichalec/hg2g/apps/npm/lib/node_modules/classeur-downloader/node_modules/restler/lib/restler.js:198:29)
    at /home/pmichalec/hg2g/apps/npm/lib/node_modules/classeur-downloader/node_modules/restler/lib/restler.js:157:16
    at Request.mixin._decode (/home/pmichalec/hg2g/apps/npm/lib/node_modules/classeur-downloader/node_modules/restler/lib/restler.js:173:7)
```

part of the options:

```
headers: 
      { Accept: '*/*',
        'User-Agent': 'Restler for node.js',
        Host: 'app.classeur.io',
        'Accept-Encoding': 'gzip, deflate',
        'x-agent-info': 'github.com/zbentley/classeur-api-client',
        Authorization: 'Basic bEpEOX..........xxxxxx..........5dVo=',
        'Content-Length': 0 },
     request: 
      ClientRequest {
        domain: null,
        _events: [Object],
        _eventsCount: 3,
        _maxListeners: undefined,
        output: [],
        outputEncodings: [],
        outputCallbacks: [],
        outputSize: 0,
        writable: true,
        _last: true,
        chunkedEncoding: false,
        shouldKeepAlive: false,
        useChunkedEncodingByDefault: false,
        sendDate: false,
        _removedHeader: [Object],
        _contentLength: 0,
        _hasBody: true,
        _trailer: '',
        finished: true,
        _headerSent: true,
        socket: [Object],
        connection: [Object],
        _header: 'GET /api/v1/folders/Salt HTTP/1.1\r\nAccept: */*\r\nUser-Agent: Restler for node.js\r\nHost: app.classeur.io\r\nAccept-Encoding: gzip, deflate\r\nx-agent-info: github.com/zbentley/classeur-api-client\r\nAuthorization: Basic bEpEOXVwZ..............xxxxxxx.............M0lSQUU5dVo=\r\nContent-Length: 0\r\nConnection: close\r\n\r\n',
        _headers: [Object],
        _headerNames: [Object],
        _onPendingData: null,
        agent: [Object],
        socketPath: undefined,
        method: 'GET',
        path: '/api/v1/folders/Salt',
        _ended: true,
        parser: null,
        res: [Object] } },
  data: 
   { status: 400,
     error: 'bad_request',
     reason: 'unexpected_request_content_type' },
  status: 400,
  reason: 'unexpected_request_content_type',
  shortReason: 'bad_request' }
```

**Comment by zbentley** [comment]
I will investigate this issue this week; if I can reproduce it I should have a fix within a day or two. Thanks for reporting!

**Comment by zbentley** [comment]
Fixed in version 0.1.3; please let me know if you experience other issues.

Thanks for your patience!
